docs/src/DeveloperDocumentation/printing_details.md

```diff
@@ -173,7 +175,7 @@ end
 
 function Base.show(io::IO, b::B)
   io = AbstractAlgebra.pretty(io)
-  print(io, LowercaseOff(), "Hilbert thing")
+  print(io, AbstractAlgebra.LowercaseOff(), "Hilbert thing")
```

Comment:
I believe this is exported? Then `AbstractAlgebra.` is not necessary.

Reply:
The example code uses an explicit `AbstractAlgebra.` everywhere else, and it should be consistent. With it there, a user can copy&paste the example code and start working with it, whether or not they have done `using AbstractAlgebra` (which is dangerous inside a package) or `import AbstractAlgebra`.

Anyway, my primary concern was to make it consistent. If someone prefers to drop the `AbstractAlgebra.` everywhere, that should be a seperate PR, IMHO